build-image.sh

```diff
@@ -153,6 +153,12 @@ fi
 chroot $chroot_dir dpkg-divert --local --rename --add /sbin/initctl
 chroot $chroot_dir ln -sf /bin/true /sbin/initctl
 
+### Build semop wrapper
+cp wrap_semop.c $chroot_dir/tmp/
+chroot $chroot_dir apt-get -y install build-essential
+chroot $chroot_dir gcc -fPIC -shared -o /opt/libpreload-semop.so /tmp/wrap_semop.c
+chroot $chroot_dir printf /opt/libpreload-semop.so > $chroot_dir/etc/ld.so.preload
```

Comment:
Nit: simple `echo` should work here.

Reply:
Echo in e41f5379a7a7af2117678ad3695ee1ceebd39d01